posthog/settings/web.py

```diff
@@ -227,3 +227,5 @@ def add_recorder_js_headers(headers, path, url):
 WHITENOISE_ADD_HEADERS_FUNCTION = add_recorder_js_headers
 
 CSRF_COOKIE_NAME = "posthog_csrftoken"
+
+GZIP_RESPONSE_ALLOW_LIST = get_list(os.getenv("GZIP_RESPONSE_ALLOW_LIST", "snapshots/?$"))
```

Comment:
Probably out of scope here, but would be nice to also gzip the frontend JS that's served to users fro apps, urls like:
```
https://app.posthog.com/api/plugin_config/1234/frontend
```

Reply:
🤔 

Where do we set env variables for production?

Reply:
I'm not sure where _secrets_ are set (I thought it's AWS Secrets Manager but seems like no more 🤔), this however is not secret, so `task-definition.web.json` is the place (though we probably want Cloud and self-hosted to behave the same way here, so maybe not needed?)

BTW, what URLs are we targeting with the `snapshots/?$` rule? Seems pretty lax, as it matches any URL ending in the string "snapshots".

Reply:
> Where do we set env variables for production?

In the task definitions at the root of the repo